src/webots/nodes/WbPositionSensor.cpp

```diff
@@ -56,8 +56,11 @@ void WbPositionSensor::init() {
   mResolution = findSFDouble("resolution");
 }
 
+void WbPositionSensor::preFinalize() {
+  WbJointDevice::preFinalize();
+}
+
```

Comment:
This is a derived method. There is no need to override it with the exact same value as in the base class.